### DISCUSSION TITLE
Referencing a preset on Azure DevOps with space in project name doesn't work

### DISCUSSION THREAD

**Discussion by kipusoep** [renovatebot/renovate#21704] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.7.2

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I found out that when I'm referencing a preset in another repository with a path, and the project name contains a space character, a "Preset is invalid" error is thrown.
This is because of this regex:
https://github.com/renovatebot/renovate/blob/2bfb5604dcb2a3eb1af67d747665caed4cd527ef/lib/config/presets/index.ts#L41
I think it should be adjusted to something like this (allow a space in the repo part):
`/^(?<repo>~?[\w\-. /]+?)\/\/(?:(?<presetPath>[\w\-./]+)\/)?(?<presetName>[\w\-.]+)(?:#(?<tag>[\w\-./]+?))?$/`

The other regex right below this one does contain the space:
https://github.com/renovatebot/renovate/blob/2bfb5604dcb2a3eb1af67d747665caed4cd527ef/lib/config/presets/index.ts#L44
That one got fixed approximately 5 months ago: https://github.com/renovatebot/renovate/commit/55d3c0468dc6ef76267e5bc9eaab548f0c5e7283

So to be clear, this is my extends config setting:
`"extends": [ "Company Name/Infrastructure//renovate/default" ]`

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Does it work if you do `"extends": [ "Company%20Name/Infrastructure//renovate/default" ]`? I'm not sure, but just trying to think of a workaround.

**Comment by kipusoep** [comment]
Yeah tried that, doesn't work because it does a string compare.
